vsphere/resource_vsphere_compute_cluster.go

```diff
@@ -527,6 +528,27 @@ func resourceVSphereComputeCluster() *schema.Resource {
 				Computed:    true,
 				Description: "Whether the vSAN unmap service is enabled for the cluster.",
 			},
+			"vsan_remote_datastore_ids": {
```

Comment:
HCI Mesh conflicts with the use of DIT, it I recall correctly. Should there be a `ConflictsWith` between these?

Reply:
Hi Ryan, thanks for the review! I have resolved your comments in the latest version. Please take another look and see if there is any open issues, thanks!